docs/guidance/EnterpriseScaleAnalytics-CostEstimates.md

```diff
@@ -6,8 +6,8 @@ This document will provide users an overview of what monthly cost can be expecte
 
 In a production scenario, it is recommended to rely on the Azure Firewall and Private DNS Zones hosted in the connectivity Hub of [Enterprise-Scale Landing Zones](https://github.com/Azure/Enterprise-Scale). In MVPs, users may rely on the Azure Firewall and Private DNS Zones bundled with Enterprise-Scale Analytics. Hence, two different cost calclations are provided which can be found below:
 
-- [Pricing Calculator - Data Management Zone w/o Azure Firewall and Private DNS Zones](https://azure.com/e/81ea206a75764d6ba390332f04d3fbe1)
-- [Pricing Calculator - Data Management Zone w/ Azure Firewall and Private DNS Zones](https://azure.com/e/7461e20e019a4d39aa653e4e516f23af)
+- [Pricing Calculator - Data Management Zone w/o Azure Firewall and Private DNS Zones](https://azure.com/e/ebb7508a6d78487b9a1583878d0948cc)
+- [Pricing Calculator - Data Management Zone w/ Azure Firewall and Private DNS Zones](https://azure.com/e/658478643d4b46fdbf8b1972c4b0704b)
 
 Please keep in mind that the following services are not listed in the overview, because they are either not available in the official Pricing Calculator or the Pricing Calculator does not allow to set the compute hours to 0:
```

Comment:
Hey Marvin, Perhaps this not clear to me but. Where is the overview you mentioned in this statement ? Should we include a link to where the overview is ? Everything else looks good and clear. 

Thanks

Reply:
The overview is the "Cost Calculator". Just click on the link. :)

Reply:
It can be just me. I was not clear to me that you were referring to the Cost Calculator. Shouldn't we say Cost Calculator  overview ? 

Thanks